editor.planx.uk/src/components/EditorNavMenu/EditorNavMenu.tsx

```diff
@@ -99,31 +100,37 @@ function EditorNavMenu() {
       title: "Settings",
       Icon: TuneIcon,
       route: `/${teamSlug}/general-settings`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
     },
     {
       title: "Design",
       Icon: PaletteIcon,
       route: `/${teamSlug}/design`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
     },
     {
       title: "Team members",
       Icon: GroupIcon,
       route: `/${teamSlug}/members`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
+    },
+    {
+      title: "Subscription",
+      Icon: CurrencyPoundIcon,
+      route: `/${teamSlug}/subscription`,
+      accessibleBy: ["platformAdmin", "teamAdmin"],
```

Comment:
The `Subscription` page simply says "Coming soon" for now and I think `accessibleBy: ["platformAdmin", "teamAdmin"]` is basically as good as feature flagged - any objections?

Reply:
Yep this is great 👍